src/main/kotlin/race/ui/InputView.kt

```diff
@@ -1,15 +1,20 @@
 package race.ui
 
+import java.lang.IllegalArgumentException
+
 class InputView {
```

Comment:
`object`를 이용해도 좋을 것 같아요 :)
```suggestion
object InputView {
```